### ISSUE TITLE
import/no-relative-parent-imports report incorrectly in 2.20.1

### ISSUE THREAD

**Issue by blacksteed232** [benmosher/eslint-plugin-import#1644]
In version `2.20.1`, I start to get errors like:

```
Relative imports from parent directories are not allowed. Please either pass what ...
```

for 

```javascript
import React from 'react';
```

The problem did not occur in `2.20.0`.

I use `eslint-typescript` but since this used to work in `2.20.0` I doubt that this has anything to do with that.

**Comment by ljharb** [comment]
What's your tsconfig look like?

**Comment by blacksteed232** [comment]
This is my `tsconfig.json`

```json
{
	"compilerOptions": {
		"target": "esnext",
		"lib": [
			"dom",
			"dom.iterable",
			"esnext",
			"webworker",
			"scripthost"
		],
		"skipLibCheck": true,
		"esModuleInterop": true,
		"experimentalDecorators": true,
		"allowSyntheticDefaultImports": true,
		"strict": true,
		"forceConsistentCasingInFileNames": true,
		"module": "esnext",
		"moduleResolution": "node",
		"resolveJsonModule": true,
		"isolatedModules": true,
		"noEmit": true,
		"jsx": "preserve",
		"baseUrl": ".",
		"paths": {
			// some module paths, for example "utils": ["./app/utils]
		}
	},
	"include": [
		"./app/**/*.tsx",
		"./app/**/*.ts"
	],
	"exclude": [
		"./app/index.tsx"
	]
}
```

**Comment by ljharb** [comment]
@blacksteed232 can you check and see if this still occurs on latest master?

**Comment by blacksteed232** [comment]
@ljharb hey would love to, but could you let me know roughly how I can package this and install it locally?

**Comment by ljharb** [comment]
@blacksteed232 temporarily you should be able to `npm install --no-save benmosher/eslint-plugin-import`

**Comment by blacksteed232** [comment]
@ljharb yikes, really sorry, I'm behind a firewall so I cannot do this :(

**Comment by ljharb** [comment]
I'm a bit confused; if you can post on github, surely you can download from it?

**Comment by blacksteed232** [comment]
ah, that codebase is on a remote machine :D. I will spend some time try to setup a repo and test this out later

**Comment by blacksteed232** [comment]
@ljharb so I have tried to recreate the problem on MacOS and there was no problem. But the first time I saw the problem was on a Windows machine at work so I'll try that and let you know.

**Comment by blacksteed232** [comment]
@ljharb seems like the problem still shows up on Windows :(

**Comment by ljharb** [comment]
@blacksteed232 any update?

**Comment by blacksteed232** [comment]
@ljharb yeah I tried the one on master but the problem still occured

**Comment by Heliks** [comment]
Having the same issue with `reflect-metadata`

**Comment by ljharb** [comment]
@Heliks can you elaborate?

**Comment by Heliks** [comment]
@ljharb Same problem as described in the initial issue. The line `import 'reflect-metadata';` is reported on windows systems while it works fine with linux.

**Comment by FRiviere** [comment]
Having the same issue with `import Vue from "vue"` only try on windows
